### ISSUE TITLE
Using Promise in a flint app will crash it, but using window.Promise will not.

### ISSUE THREAD

**Issue by AriaFallah** [flintjs/flint#285]
Steps to reproduce:

`flint new <app>`

```javascript
// main.js
console.log(Promise) // crashes app
console.log(window.Promise) // works fine
```

**Comment by ncammarata** [comment]
![image](https://cloud.githubusercontent.com/assets/204989/12546495/048926ac-c2ff-11e5-9a67-0205878441b1.png)

We aren't loading in the babel-runtime promise correct. Errors on this

**Comment by natew** [comment]
Should be fixed in 1.4.71
